src/mono/wasm/build/WasmApp.targets

```
@@ -486,7 +486,20 @@
 
     <Message Text="Compiling native assets with emcc. This may take a while ..." Importance="High" />
     <Exec Command='emcc $(EmccCFlags) "%(_WasmObjectsToBuild.SourcePath)" -c -o "%(_WasmObjectsToBuild.Identity)"' EnvironmentVariables="@(EmscriptenEnvVars)" />
-    <Exec Command="emcc $(EmccLDFlags) @(_DotnetJSSrcFile->'--js-library &quot;%(Identity)&quot;', ' ') @(_BitcodeFile->'&quot;%(Identity)&quot;', ' ') @(_WasmObjects->'&quot;%(Identity)&quot;', ' ') -o &quot;$(_WasmIntermediateOutputPath)dotnet.js&quot;" EnvironmentVariables="@(EmscriptenEnvVars)" />
+
+    <PropertyGroup>
+      <_EmccLinkerArguments>$(EmccLDFlags)</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) @(_DotnetJSSrcFile->'--js-library &quot;%(Identity)&quot;', ' ')</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) @(_BitcodeFile->'&quot;%(Identity)&quot;', ' ')</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) @(_WasmObjects->'&quot;%(Identity)&quot;', ' ')</_EmccLinkerArguments>
+      <_EmccLinkerArguments>$(_EmccLinkerArguments) -o &quot;$(_WasmIntermediateOutputPath)dotnet.js&quot;</_EmccLinkerArguments>
+
+      <_EmccLinkerResponseFile>$(_WasmIntermediateOutputPath)emcc-link.rsp</_EmccLinkerResponseFile>
+    </PropertyGroup>
+    <WriteLinesToFile Lines="$(_EmccLinkerArguments)" File="$(_EmccLinkerResponseFile)" />
+    <Message Text="Running emcc with $(_EmccLinkerArguments)" Importance="Low" />
+    <Exec Command="emcc @$(_EmccLinkerResponseFile)" EnvironmentVariables="@(EmscriptenEnvVars)" />
```

Super nit, I would consider using `%40` here since `@` has special semantics in MSBuild
```suggestion
    <Exec Command="emcc %40$(_EmccLinkerResponseFile)" EnvironmentVariables="@(EmscriptenEnvVars)" />
```